libraries/apollo-tooling/api/apollo-tooling.api

```diff
@@ -37,15 +37,6 @@ public final class com/apollographql/apollo3/tooling/PublishOperationsSuccess :
 	public final fun getUpdated ()I
 }
 
-public final class com/apollographql/apollo3/tooling/SchemaDownloader$SpecVersion : java/lang/Enum {
```

Comment:
Technically breaking! I think it was public by mistake.

Reply:
I think there is some value in allowing the user to get an introspection query for a subset of supported features. Spec version dates is probably the bad abstraction (`@oneOf` being an example) but allowing to pass options like [graphql-js is doing ](https://github.com/mjmahone/graphql-js/blob/8724e22f9e89224dd2669d6fc0c4b7fe34c2c8dd/docs-old/APIReference-Utilities.md#L111) would be nice.

Reply:
In 75b0843611bb789b6cac6bfd4123ee995351e3d1 I made `IntrospectionCapability` public (and `@ApolloExperimental`) and added `SchemaDownloader.getIntrospectionQuery(capabilities: Set<IntrospectionCapability>): String`